File: presenters/src/main/java/ro/upt/sma/heart/presenters/assistant/HeartAssistantView.java
package ro.upt.sma.heart.presenters.assistant;

import java.util.List;
import ro.upt.sma.heart.model.HeartMeasurement;

public interface HeartAssistantView {

  void showLast(HeartMeasurement heartMeasurement);

  void showList(List<HeartMeasurement> heartMeasurementList);

}

File: presenters/src/main/java/ro/upt/sma/heart/presenters/assistant/HeartAssistantView.kt
package ro.upt.sma.heart.presenters.assistant

import ro.upt.sma.heart.model.HeartMeasurement

interface HeartAssistantView {
    fun showLast(heartMeasurement: HeartMeasurement)
    fun showList(heartMeasurementList: List<HeartMeasurement>)
}
